### PR TITLE
[DOCFIX] Update doc to deprecate loadMetadata command.

### PR DESCRIPTION
**Pull request by ifcharming**

### CODE REVIEW

File: docs/_data/table/en/operation-command.yml [hunk 1]
```diff
@@ -28,6 +28,8 @@ getUsedBytes:
 load:
   Load the data of a file or a directory from under storage into Alluxio.
 loadMetadata:
+  The loadMetadata command is deprecated since Alluxio version 1.1.  Please use "alluxio fs ls
+  <path>" command instead.
```
Comment (calvinjia) [inline docs/_data/table/en/operation-command.yml]: Should we update the documentation of `ls`? Just looking at this it is unclear how I would use `ls` to replace the functionality of `loadMetadata`

Comment (ifcharming) [inline docs/_data/table/en/operation-command.yml]: Good point. Updated the `ls` description, thanks.

**Comment by ifcharming** [comment]
thanks @calvinjia for taking a look. PTAL

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**